bin/morgue.js

```diff
@@ -7461,7 +7471,22 @@ function main() {
       errx("Unable to read configuration: " + err.message + ".");
     }
 
-    command(argv, config);
+    /*
```

Comment:
I'd split out that change in a different commit; it'll be easier to describe that way too.